### PR TITLE
khard: 0.18.0 -> 0.19.0

### PR DESCRIPTION
**Pull request by antonmosich**
## Description of changes
See https://github.com/lucc/khard/blob/v0.19.0/CHANGES#L4-L16 for the changelog

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### Priorities

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]

**Review by matthiasbeyer — APPROVED** [review]

**Comment by matthiasbeyer** [comment]
Result of `nixpkgs-review pr 269609` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>khard</li>
    <li>khard.dist</li>
    <li>lbdb</li>
  </ul>
</details>

File: pkgs/applications/misc/khard/default.nix [hunk 1]
```diff
@@ -1,12 +1,14 @@
-{ lib, python3, fetchPypi, khard, testers }:
+{ lib, python3, fetchPypi, khard, testers, installShellFiles }:
 
 python3.pkgs.buildPythonApplication rec {
-  version = "0.18.0";
+  version = "0.19.0";
   pname = "khard";
+  format = "pyproject";
```
Comment (natsukium) [inline pkgs/applications/misc/khard/default.nix:6]:
```suggestion
  pyproject = true;
```
Could you please use a parameter, `pyproject`, instead?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

File: pkgs/applications/misc/khard/default.nix [hunk 2]
```diff
@@ -15,6 +17,7 @@ python3.pkgs.buildPythonApplication rec {
     sphinxHook
```
Comment (natsukium) [inline pkgs/applications/misc/khard/default.nix:17]:
```suggestion
    setuptools
    sphinxHook
```
We set `pyproject = true` (`format = "pyproject"`), so setuptools is no longer included in nativeBuildInputs.
Could you please add it explicitly?
https://github.com/lucc/khard/blob/8d841537b5cedc0817adcd49560978f3a546deb1/pyproject.toml#L46

**Comment by antonmosich** [comment]
As #270156 has been merged, I will close this PR.

**Pull request closed without merging**